stm32f0_hal/examples/uart.rs

```diff
@@ -1,23 +1,46 @@
 #![no_std]
 
+extern crate cortex_m;
 extern crate stm32f0_hal as hal;
+#[macro_use(interrupt)]
+extern crate stm32f0x2;
```

Comment:
For me you should not have to include this in hal examples.
I think hal is created to  hide stm32f0x2...
@pierre-rouanet any comment?

Reply:
Agreed: it's now tracked in #25